### PR TITLE
google_bigquery_connection - add missing output fields

### PR DESCRIPTION
**Pull request by DrFaust92**
If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
bigquery: added `cloud_sql.service_account_id` and `azure.identity` output fields
```

```release-note:enhancement
bigquery: added `cloud_spanner.use_serverless_analytics` field
```

### CODE REVIEW

File: mmv1/products/bigqueryconnection/api.yaml [hunk 1]
```diff
@@ -183,6 +192,9 @@ objects:
           - !ruby/object:Api::Type::Boolean
             name: 'useParallelism'
             description: If parallelism should be used when reading from Cloud Spanner
+          - !ruby/object:Api::Type::Boolean
+            name: 'useServerlessAnalytics'
```
Comment (rileykarson) [inline mmv1/products/bigqueryconnection/api.yaml:196]: Do you mind adding a test or updating a test to use this value? Thanks!

Comment (DrFaust92) [reply mmv1/products/bigqueryconnection/api.yaml:196]: sure, will do

Comment (DrFaust92) [reply mmv1/products/bigqueryconnection/api.yaml:196]: done

**Comment by DrFaust92** [comment]
@rileykarson any movement on this?

**Review by rileykarson — APPROVED** [review]
Oh, I'm sorry! I think I missed the email when you updated before. My inbox has been a little out of control, sorry about that. Please feel free to reach out sooner!

_Generally_ if it's been a week a notification got lost somewhere, so [our robot recommends](https://github.com/GoogleCloudPlatform/magic-modules/pull/6988#issuecomment-1356836453) that as a guideline.

**Comment by DrFaust92** [comment]
No worries, thanks for pushing this!

**Pull request merged**